packages/vendor/scripts/deploy-docs.sh

```diff
@@ -1,6 +1,10 @@
 #! /bin/bash
 # the docs folder need to be build before this script can run
 # use npm run build:docs
+if [ -n "${SKIP_DOCS}" ] && [ "${SKIP_DOCS}" == "true" ]
```

Comment:
Won't it make more sense to use the `TRAVIS_CI` environment variable instead?
So we only run it in travis